### PR TITLE
Style rememberchoice form

### PR DESCRIPTION
**Pull request by Badlapje**
After conferring with Arnout and Edwin, a spot was chosen for the rememberchoice feature.  This PR moves the form to that spot (between search and idp-list) & adds some styles for it.  The styles are a proposal.

### CODE REVIEW

File: theme/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "create-theme": "node scripts/create-theme.js",
     "watch:js": "watchify ${EB_THEME:=base}/javascripts/application.js --poll=100 --debug -t [ babelify ] ${EB_THEME:=base}/javascripts/application.js -o ../web/javascripts/application.min.js -v",
     "watch:css": "node-sass -r ${EB_THEME:=openconext}/stylesheets -w -o ../web/stylesheets",
+    "watch:css:single": "node-sass -r ${EB_THEME:=openconext}/stylesheets -o ../web/stylesheets",
```
Comment (MKodde) [inline theme/package.json:32]: I see that you like this! However, the name does not sit well with me. The behavior I see is a quick compilation of the scss/sass code to the web/stylesheets folder. It does not actually watch for changes. 

Maybe you where looking for: `npm run build:css:sass` ?

**Review by MKodde — COMMENTED** [review]
Nice work again! See feedback below. The remember choice feature looked good and functioned as expected with the keyboard and mouse.

Would adding an e2e test for this be worth it?

File: theme/package.json (hunk 1 above)
Comment (Badlapje) [reply theme/package.json]: gah, can't believe i missed that.  Removed it again 🤦

Comment (MKodde) [reply theme/package.json]: Not a problem!

**Review by MKodde — APPROVED** [review]

**Pull request merged**